forage-android/src/test/java/com/joinforage/forage/android/vault/BasisTheoryPinSubmitterTest.kt

```diff
@@ -52,7 +52,7 @@ class BasisTheoryPinSubmitterTest() : MockServerSuite() {
         mockBasisTheoryTextElement = mock(TextElement::class.java)
 
         // ensure we don't make any live requests!
-        mockBasisTheoryResponse(Result.success("success"))
+        mockBasisTheoryResponse(Result.success(mapOf("mySuccessfulKey" to "mySuccessfulValue")))
```

Comment:
Updating because BT starts returning Gson representation of JSON instead of just the JSON string